### PR TITLE
Develop react_tabs

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
[DEMO LINK](https://sergey-lipikhin.github.io/react_tabs/)

### CODE REVIEW

**Review by mstrilec — APPROVED** [review]
GJ!

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -1 +1,57 @@
...
+    onTabSelected,
+  } = props;
+
+  const handleTabSelect = (newTab: Tab): void => {
+    if (selectedTabId !== newTab.id) {
+      onTabSelected(newTab);
+    }
+  };
+
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
```
Comment (illiarerun) [inline src/components/Tabs/Tabs.tsx:24]: you repeat this logic in App.tsx - create a function helper for it

**Review by illiarerun — CHANGES_REQUESTED** [review]

File: src/App.tsx [hunk 2]
```diff
@@ -1,42 +1,38 @@
...
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
+import { Tab } from './types/Tab';
+import { Tabs } from './components/Tabs';
+import { getSelectedTab } from './helpers';
+
 export const tabs = [
```
Comment (sergii-nosachenko) [inline src/App.tsx:10]:
```suggestion
export const tabs: Tab[] = [
```

File: src/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { Tab } from './types/Tab';
+
+export function getSelectedTab(tabs: Tab[], selectedTabId: string): Tab {
```
Comment (sergii-nosachenko) [inline src/helpers.ts:3]:
```suggestion
export function getSelectedTabById(tabs: Tab[], selectedTabId: string): Tab {
```

**Review by sergii-nosachenko — APPROVED** [review]
Good work 👍
Commented

**Pull request closed without merging**